compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -516,6 +527,27 @@ object Completion:
           .toSeq
           .groupByName
 
+    /** Completions for named tuples */
+    private def namedTupleCompletions(qual: tpd.Tree)(using Context): CompletionMap =
+      def namedTupleCompleionsFromType(tpe: Type): CompletionMap =
+        tpe.namedTupleElementTypes
```

Comment:
Shouldn't we create a new fresh context for this ?

Reply:
Probably, it will definitely be safer to do so. Good point.